### DISCUSSION TITLE
Is it possible to Replace TemplateUserLostPassword.cshtml?

### DISCUSSION THREAD

**Discussion by XpandIT-RFID** [OrchardCMS/OrchardCore#10199] [Q&A] [answered]
Hi, I need to replace the TemplateUserLostPassword.cshtml to use my own, is there an easy way of doing this or do I need to have the module on my project to change the code?

https://github.com/OrchardCMS/OrchardCore/blob/339d74b251b53393dd510f8ad3027aa17d78500d/src/OrchardCore.Modules/OrchardCore.Users/Views/TemplateUserLostPassword.cshtml

Cheers,
Ricardo Duarte

**Comment by Skrypt** [comment, marked as answer]
Yes, you can override these views in a custom module or in a custom theme. They would require to have a manifest.cs dependency on the OrchardCore.Users module to be able to override it's views. Also of course a csproj package reference to the OrchardCore.Users module.

**Reply by XpandIT-RFID** [reply]
I was trying to do this with the placements.json XD, these overrides and such need some documentation :/ It is a little confusing without knowing these tricks in my opinion.

Thanks for the help @Skrypt
